New commands for "Microsoft Graph Mail" Integration (#44324)
demisto/content: Packs/ApiModules/ReleaseNotes/2_4_17.md

#### Scripts

##### MicrosoftGraphMailApiModule

> Added support for the ***msgraph-mail-create-rule***, ***msgraph-mail-update-rule***, ***msgraph-mail-get-settings***, and ***msgraph-mail-get-mailtips*** commands, used by the Microsoft Graph Mail integration.

Reviewer: ```suggestion
Added support for the ***msgraph-mail-create-rule***, ***msgraph-mail-update-rule***, ***msgraph-mail-get-settings***, and ***msgraph-mail-get-mailtips*** commands in the Microsoft Graph Mail integration.
```
